pkg/names/names_test.go

```
@@ -0,0 +1,43 @@
+package names
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNamespaceDefault(t *testing.T) {
+	// When OPERATOR_NAMESPACE is not set, Namespace should be "openshift-ptp"
+	os.Unsetenv("OPERATOR_NAMESPACE")
```

golangci-lint (2.12.2)

[error] Error return value of `os.Unsetenv` is not checked

(errcheck)